### PR TITLE
Fixing className var initialization

### PR DESCRIPTION
**Pull request by mourasman**
className was being initialized in the wrong place. sorry about that! 😅

### CODE REVIEW

**Comment by millmanorama** [comment]
This looks like a good fix, is there any plan to merge it?

**Review by znevrly — APPROVED** [review]

**Comment by domhall** [comment]
+1 any chance of merging this? Looks like the package version number got updated but the PR didn't get merged by mistake maybe?

Possibly should consider running tests in a pre-merge github action

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cucumber-junit-convert",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (domhall) [inline package.json:3]:
```suggestion
  "version": "2.2.1",
```
Package is now on version 2.2.0 for some reason

**Comment by znevrly** [comment]
Sorry, I publish it twice with increased version. I will resolve conflict and publish it.

**Pull request merged**